pkgdb2client.py

```diff
@@ -0,0 +1,25 @@
+import pkgdb2client
+import json
+
+def get_packages(name):
```

Comment:
Can you create new class ``PkgdbClient`` with getPackages(name) method? the module can contain more methods in future and it is easier to replace object with another object with the same interface.

I have already PackageManager class [1] which I could use instead of PkgdbClient for testing purpose. Or I could introduce FakePkgdbClient what would return predefined list of packages. Or create CachedPkgdbClient in order to gain some speed.

[1] https://github.com/gofed/lib/blob/master/packagemanager.py